RCTVideo.xcodeproj/project.pbxproj

```diff
@@ -222,8 +222,8 @@
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
-					"$(SRCROOT)/../../React/**",
-					"$(SRCROOT)/../react-native/React/**",
+					"$(SRCROOT)/../../React",
+					"$(SRCROOT)/../react-native/React",
```

Comment:
See http://stackoverflow.com/questions/3490629/error-argument-list-too-longrecursive-header-expansion-failed-at-application
It's still not pretty: I have to locally install `npm react-native` while leaving `package.json` unmodified.